index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title>Portfolio: Setup and Mobile First</title>
+</head>
+<body>
+    <main class="header-background-mobile">
+        <header class="toolbar">
+            <nav>
+                <div class="mylogo">
+                    <a href="#">My Logo</a>
+                </div>
+                <div class="union">
+                    <img src="./images/Union.png" alt="Hamburguer Menu">
+                </div>
+            </nav>
+        </header>
```

Comment:
- [ ] Kindly make sure the content of the `header` is aligned with `flexbox` as required for the project.